backend/api/Database/Models/Mission.cs

```diff
@@ -149,6 +151,22 @@ public void CalculateEstimatedDuration()
             );
             EstimatedDuration = TimeSpan.FromMinutes(estimate);
         }
+
+        public void SetToFailed()
+        {
+            Status = MissionStatus.Failed;
+            StatusReason = "Lost connection do ISAR during mission";
```

Comment:
Typo: should be "StatusReason = "Lost connection **to** ISAR during mission";"

Reply:
Good catch!